CallsheetLocalizations/es-MX.xcloc/Localized Contents/es-MX.xliff

```diff
@@ -434,6 +434,7 @@
       </trans-unit>
       <trans-unit id="Dark" xml:space="preserve">
         <source>Dark</source>
+        <target state="translated">Oscuro</target>
```

Comment:
Both **Oscuro** and **Obscuro** are valid. However, Apple docs in ES show it as "Obscuro"
https://support.apple.com/es-us/guide/iphone/iph60ba71065/ios

Reply:
85139fb4c4ef5866a3af74013c4aa8130e08481b